src/main/flight/gps_rescue.c

```diff
@@ -138,7 +138,8 @@ PG_RESET_TEMPLATE(gpsRescueConfig_t, gpsRescueConfig,
     .throttleHover = 1280,
     .sanityChecks = RESCUE_SANITY_ON,
     .minSats = 8,
-    .minRescueDth = 100
+    .minRescueDth = 100,
+    .allowArmingWithoutFix = 0,
```

Comment:
If this is a true-false, then is more clear to use that, for example in `pid.c`:
```
        .launchControlAllowTriggerReset = true,
```
If you plan in a future to have more states, then an enum will be ok.

Reply:
Sorry, the uint8_t declaration sidetracked me O:-)